ocp4/profiles/coreos-ncp.profile

```diff
@@ -140,75 +133,75 @@ selections:
 
     ### Kernel Config
     ## Boot prompt
-    #- grub2_audit_argument
-    #- grub2_audit_backlog_limit_argument
-    #- grub2_slub_debug_argument
-    #- grub2_page_poison_argument
-    #- grub2_vsyscall_argument
-    #- grub2_vsyscall_argument.role=unscored
-    #- grub2_vsyscall_argument.severity=info
-    #- grub2_pti_argument
+    - grub2_audit_argument
+    - grub2_audit_backlog_limit_argument
+    - grub2_slub_debug_argument
+    - grub2_page_poison_argument
+    - grub2_vsyscall_argument
+    - grub2_vsyscall_argument.role=unscored
+    - grub2_vsyscall_argument.severity=info
+    - grub2_pti_argument
 
     ## Security Settings
-    #- sysctl_kernel_kptr_restrict
-    #- sysctl_kernel_dmesg_restrict
-    #- sysctl_kernel_kexec_load_disabled
-    #- sysctl_kernel_yama_ptrace_scope
-    #- sysctl_kernel_perf_event_paranoid
-    #- sysctl_user_max_user_namespaces
-    #- sysctl_user_max_user_namespaces.role=unscored
-    #- sysctl_user_max_user_namespaces.severity=info
-    #- sysctl_kernel_unprivileged_bpf_disabled
-    #- sysctl_net_core_bpf_jit_harden
+    - sysctl_kernel_kptr_restrict
+    - sysctl_kernel_dmesg_restrict
+    - sysctl_kernel_kexec_load_disabled
+    - sysctl_kernel_yama_ptrace_scope
+    - sysctl_kernel_perf_event_paranoid
+    - sysctl_user_max_user_namespaces
+    - sysctl_user_max_user_namespaces.role=unscored
+    - sysctl_user_max_user_namespaces.severity=info
```

Comment:
Does this rule make sense for RHCOS since its only use-case is to run containers?

Reply:
Actually, upon reading the rule, it seems just informational?

Reply:
If a configuration option could be set then it must be evaluated, regardless of the intended product use case.